### DISCUSSION TITLE
unsuccessful upgrade from 2.8.7 to 2.11.4

### DISCUSSION THREAD

**Discussion by eudjinnl** [netbox-community/netbox#6494] [Q&A] [answered]
Tried to upgrade via option B from https://netbox.readthedocs.io/en/stable/installation/upgrading/
Fortunately it was a test server :)

Now I have error:

Server Error
There was a problem with your request. Please contact an administrator.

The complete exception is provided below:

<class 'django.contrib.contenttypes.models.ContentType.DoesNotExist'>

ContentType matching query does not exist.

Python version: 3.8.5
NetBox version: 2.11.4
If further assistance is required, please post to the NetBox mailing list.

**Comment by candlerb** [comment, marked as answer]
https://groups.google.com/g/netbox-discuss/search?q=django.contrib.contenttypes.models.ContentType.DoesNotExist

Almost certainly you forgot to run the `./upgrade.sh` script; or if you did, it failed, in which case we need to see the error which the script reported.

**Reply by eudjinnl** [reply]
My bad.

I had run the script. There were error about reddis, I fixed it but didn't start upgrade script again.

Thank you!
